src/components/Challenges.js

```diff
@@ -1,16 +1,108 @@
-import React from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import React from 'react';
+import {Col, Container, Row, Carousel} from 'react-bootstrap';
+import GreenPeas from "../assets/greenpeas";
+import OnionCorn from "../assets/OnionCorn";
+import CelebratoryBoi from "../assets/CelebratoryBoi";
+import BeanstalkBoi from "../assets/BeanstalkBoi";
+const Challenges = () => {
+  return (
+    <div>
+      <Container className="mt-5" >
+        <Row>
+          <Col>
+            <h1 id="challenges" className="font-weight-bold">Tech Challenges</h1>
 
-const Challenges = () => (
-  <Container className="mt-5">
-    <Row>
-      <Col>
-        <h1 id="challenges" className="font-weight-bold">
-          Tech Challenges
-        </h1>
-      </Col>
-    </Row>
-  </Container>
-);
+          </Col>
+        </Row>
+      </Container>
+      <Carousel interval={null} touch={true}>
+        <Carousel.Item>
+          <div className="carouselItem">
+            <Row>
+              <Col className="carouselImage">
+                <GreenPeas id="GreenPeas"/>
+              </Col>
+              <Col>
+                <div className="carouselHeader">
+                  <h2>Camp HackBeanpot Spirit Award:</h2>
+                </div>
+                <div className="carouselCaption">
+                  <p>After a weekend of completing challenges with your Cabins and making projects, we are so excited to see what you’ve learned at Camp HackBeanpot.
+                    We challenge you to build something that embodies our values of exploration and teamwork.
+                    This could involve a project to express your gratitude to nature or a survival tool for you and your fellow hackers.
+                  </p>
+                </div>
+              </Col>
+            </Row>
+          </div>
+        </Carousel.Item>
+
+        <Carousel.Item>
+          <div className="carouselItem">
+            <Row>
+              <Col className="carouselImage">
+                <OnionCorn id="OnionCorn"/>
+              </Col>
+              <Col>
+                <div className="carouselHeader">
+                  <h2>Most Inclusive Hack:</h2>
+                </div>
+                <div className="carouselCaption">
+                  <p>Last year was full of social change, and it doesn’t need to stop there! We challenge you to continue the trend into 2021 by creating a hack that uses tech to promote social good, accessibility, or impact on the world around you. <br/>
+                    - Did you know there <a href="https://www.w3.org/TR/wai-aria-1.1/">is a spec for web accessibility?</a> <br/>
```

Comment:
these links are hard to see :(

Reply:
It looks like the designs in Figma use `#6499C6` as the color for the links and it's a bit more visible. Maybe you could switch to using that?

Reply:
Just updated the link color 👍